session.html

```diff
@@ -18,12 +18,20 @@
     <title>MyMood | Session</title>
     
     <style>
-    .back{
+    .back {
       font-size: 20px;
       position: fixed;
       margin-left: 13%;
       margin-top: 10px;
     }
+    
+    th {
```

Comment:
th and zebra-stripe are defined twice: once in database.html and once in session html. but they're the same. would it be better to put these styles in main.css? since you're loading it anyways.

Reply:
might as well bring the back button there too, it looks reuseable even though we're only using it once for now